pages/product/[code].js

```diff
@@ -38,28 +110,30 @@ export default function DetailPage() {
 
   function BackToScanner() {
     return (
-      <button
-        onClick={() => {
-          router.push("/barcodeScanner");
-        }}
-      >
-        <span>◀︎</span>
-      </button>
+      <StyledBackButton>
+        <span
+          onClick={() => {
+            router.push("/barcodeScanner");
+          }}
+        >
+          <SVGIcon variant="back" width="50px" />
+        </span>
+      </StyledBackButton>
```

Comment:
interactive elements should always be buttons. Only buttons should have an onClick event.

```suggestion
      <StyledBackButton onClick={() => {
            router.push("/barcodeScanner");
          }}>
          <SVGIcon variant="back" width="50px" />
      </StyledBackButton>
```